CHANGELOG.md

```diff
@@ -1,5 +1,271 @@
+- Introduce API to allow applications to register `TurboModuleManagerDelegates` with `ReactInstanceManager` ([eb7e89e286](https://github.com/facebook/react-native/commit/eb7e89e2864e941b4a21d55a7403a6028e9a26a2) by [@RSNara](https://github.com/RSNara))
+- Added convenience methods to simplify native Event classes and ease migrations ([72d0ddc16f](https://github.com/facebook/react-native/commit/72d0ddc16f2f631003c3486e0a59e50c145ec613) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Add `onFocus` and `onBlur` for Pressable on Android. ([cab4da7288](https://github.com/facebook/react-native/commit/cab4da728814bf9d3c0cc7c9921e982bfc090730))
+
+#### iOS specific
+
+- Allow for configuring the `NSURLSessionConfiguration` ([58444c74f5](https://github.com/facebook/react-native/commit/58444c74f5c18b74e88a6c1cd0f059fe434c1a21) by [@hakonk](https://github.com/hakonk))
+- Expose `UIDatePickerStyles` as a prop for `DatePickerIOS` ([266b21baf3](https://github.com/facebook/react-native/commit/266b21baf35e052ff28120f79c06c4f6dddc51a9), [2b62e19a78](https://github.com/facebook/react-native/commit/2b62e19a78d873680b821e6d78d3ff9e94c3b1ae))
+- Adds an ability to retrieve the notifications authorization status from JavaScript side. ([b86e52a9ec](https://github.com/facebook/react-native/commit/b86e52a9ec9ec828388eb4a717a3782a54c7b3d9))
+- Allow `RCTRootView` to be initialized with a frame ([00bc09c8f7](https://github.com/facebook/react-native/commit/00bc09c8f76879eb1f9a92a6a643191da9355de8) by [@appden](https://github.com/appden))
+- Use react-native-codegen in iOS app template ([e99b8bbb40](https://github.com/facebook/react-native/commit/e99b8bbb404f8cd1f11b6c7998083be530d7b8a4) by [@hramos](https://github.com/hramos))
+- Added reset method to `RCTFabricSurface` to help with reloads ([53858ceaa3](https://github.com/facebook/react-native/commit/53858ceaa3beab02726b1bd6e125e506477d445e) by [@PeteTheHeat](https://github.com/PeteTheHeat))
+- High contrast dynamic color options for dark and light mode. ([4b9d9dda27](https://github.com/facebook/react-native/commit/4b9d9dda270acd4e0314f40490c699ffd0f6e30e) by [@birkir](https://github.com/birkir))
+
+### Changed
+
+- Update Flipper to 0.91.1, fixed iOS build support for i386, `use_flipper!()` will no longer need custom overrides to build with XCode 12.5 ([4246c75d0d](https://github.com/facebook/react-native/commit/4246c75d0d5b9dccbe0fd5ecec66b4cc0331f815) by [@mweststrate](https://github.com/mweststrate))
+- Make the RootTag an opaque type ([46ffe84453](https://github.com/facebook/react-native/commit/46ffe84453aaf3836da6abfe4c0b078ca0ac4f21) by [@nadiia](https://github.com/nadiia))
+- Convert require statements to use import from in Libraries/Components ([0afd71a18d](https://github.com/facebook/react-native/commit/0afd71a18d19edef3d6603cabd2672ffde5ba30e) by [@kacieb](https://github.com/kacieb))
```

Comment:
can delete internal refactors

Reply:
Looks like there might be a lot of those in this section, I won't comment on each